app/dashboard/page.tsx

```diff
@@ -21,6 +32,35 @@ const Dashboard: FC = () => {
     }
   };
 
+  const handleSendPayment = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    try {
+      const formErrors = paymentFormValidation.isFormDataValid(
+        paymentData,
+        balance
+      );
+      setFormError(formErrors);
+      if (Object.keys(formErrors).length > 0)
+        throw new Error("Invalid form data");
```

Comment:
I think this could be in another function.

Reply:
I have moved this logic into a new function called **_validatePaymentForm_**.